app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect, useCallback } from "react";
+import { PageLoader } from "neetoui";
+import { Header, SubHeader } from "neetoui/layouts";
+import EmptyState from "components/Common/EmptyState";
+import EmptyNotesListImage from "images/EmptyNotesList";
+
+import ContactTable from "./ContactTable";
+import { contacts as mockContacts } from "common/mock-data";
+import DeleteAlert from "./DeleteAlert";
```

Comment:
Please sort import statements such that all aliased and package imports come on top and then relative imports.

Please refer [sorting import statements](https://bigbinary.com/react-best-practices/sort-import-statements#sorting-import-statements) for more details